src/walletFactory.ts

```diff
@@ -1,10 +1,10 @@
-import { TransactionLimits, yamlConfig } from "./config"
+import { getUserWalletConfig, yamlConfig } from "./config"
```

Comment:
yamlConfig use for funder is in the scope of this PR?

Reply:
Yeah I’d consider that out of scope for this. Arvin is breaking the latter config refactoring into small steps.